### PR TITLE
AAP-32980 Corrected broken links and incorrect titles in the Operating AAP guide

### PR DESCRIPTION
**Pull request by jself-sudoku**
Corrections to links in two modules in the 2.5 [Operating AAP guide](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/operating_ansible_automation_platform/index)

### CODE REVIEW

File: downstream/modules/platform/con-why-automation-mesh.adoc [hunk 1]
```diff
@@ -16,6 +16,6 @@ The {AutomationMesh} component of the {PlatformName} simplifies the process of d
 
 * For information about automation mesh and the various ways to design your automation mesh for your environment:
 
-** For a VM-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_automation_mesh_guide_for_vm-based_installations/index[{PlatformName} {AutomationMesh} guide for VM-based installations].
+** For a VM-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_mesh_for_vm_environments/index[{PlatformName} {AutomationMesh} guide for VM-based installations].
```
Comment (dcdacosta) [inline downstream/modules/platform/con-why-automation-mesh.adoc:19]: The title of this document is now "Automation mesh for VM environments" and Aine set up attributes for URLs and links, suggest changing this to use that attribute:

link:{LinkAutomationMesh}

File: downstream/modules/platform/con-why-automation-mesh.adoc [hunk 2]
```diff
@@ -16,6 +16,6 @@ The {AutomationMesh} component of the {PlatformName} simplifies the process of d
 
 * For information about automation mesh and the various ways to design your automation mesh for your environment:
 
-** For a VM-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_automation_mesh_guide_for_vm-based_installations/index[{PlatformName} {AutomationMesh} guide for VM-based installations].
+** For a VM-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_mesh_for_vm_environments/index[{PlatformName} {AutomationMesh} guide for VM-based installations].  
 
-** For an operator-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_automation_mesh_for_operator-based_installations/index[{PlatformName} {AutomationMesh} for operator-based installations].
+** For an operator-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_mesh_for_managed_cloud_or_operator_environments/index[{PlatformName} {AutomationMesh} for operator-based installations].
```
Comment (dcdacosta) [inline downstream/modules/platform/con-why-automation-mesh.adoc:21]: The title of this document is now "Automation mesh for managed cloud or operator environments" and Aine set up attributes for URLs and links, suggest changing this to use that attribute:

link:{LinkOperatorMesh}

File: downstream/modules/platform/proc-change-ssl-installer.adoc [hunk 3]
```diff
@@ -9,10 +9,7 @@ The following procedure describes how to change the SSL certificate and key in t
...
-link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/index[{PlatformName} Installation Guide]
-for guidance on setting these variables.
+Refer to the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/rpm_installation/appendix-inventory-files-vars#ref-controller-variables[{ControllerNameStart} variables], link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/rpm_installation/appendix-inventory-files-vars#ref-hub-variables[{HubNameStart} variables], and link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/rpm_installation/appendix-inventory-files-vars#event-driven-ansible-controller[{EDAcontroller} variables] sections of the {LinkInstallationGuide} document for guidance on setting these variables.
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-change-ssl-installer.adoc:12]: Aine set up attributes for URLs and links, suggest changing these to use those attributes:

link:{URLInstallationGuide}/appendix-inventory-files-vars#ref-controller-variables[{ControllerNameStart} variables]

link:{URLInstallationGuide}/appendix-inventory-files-vars#ref-hub-variables[{HubNameStart} variables]

link:{URLInstallationGuide}/appendix-inventory-files-vars#event-driven-ansible-controller[{EDAcontroller} variables]

The link attribute {LinkInstallationGuide} should be changed to the title attribute {TitleInstallationGuide}

**Review by dcdacosta — COMMENTED** [review]
A few changes to use and fix attributes for URLs and links.

File: downstream/modules/platform/con-why-automation-mesh.adoc (hunk 1 above)
Comment (jself-sudoku) [reply downstream/modules/platform/con-why-automation-mesh.adoc:19]: Still getting used to this! Thanks.

File: downstream/modules/platform/con-why-automation-mesh.adoc (hunk 2 above)
Comment (jself-sudoku) [reply downstream/modules/platform/con-why-automation-mesh.adoc:21]: Done.

File: downstream/modules/platform/proc-change-ssl-installer.adoc (hunk 3 above)
Comment (jself-sudoku) [reply downstream/modules/platform/proc-change-ssl-installer.adoc:12]: Thanks! Done.

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**